Library/OcCpuLib/OcCpuLib.c

```diff
@@ -1381,6 +1403,38 @@ OcCpuScanProcessor (
     Cpu->ExtModel,
     Cpu->ExtFamily
     ));
+    
+    
+    // If we are under visualization and cpuid invtsc is enabled, we can just read 
```

Comment:
Please ensure the trailing slashes `//` are added around the comments here and inside the code below. Also turn that double space into one.